lib/clixon/clixon_xml_map.h

```diff
@@ -68,6 +77,8 @@ int assign_namespace_element(cxobj *x0, cxobj *x1, cxobj *x1p);
 int assign_namespace_body(cxobj *x0, cxobj *x1);
 int xml_merge(cxobj *x0, cxobj *x1, yang_stmt *yspec, char **reason);
 int yang_valstr2enum(yang_stmt *ytype, char *valstr, char **enumstr);
+int yang_bitsstr2val(yang_stmt *ytype, char *bitsstr, unsigned char **snmpval, size_t *snmplen);
```

Comment:
Please add a clixon_handle h as first parameter.
I see it is not strictly necessary, but I have found that it may helper later in a function's life-cycle.
For example if CLIXON_BITS_POS is changed to a clixon config option instead.
Note that in some contexts it may not be possible to pass the handle